refinery/data_set_manager/tests.py

```diff
@@ -943,12 +943,13 @@ def test_generate_solr_params_no_params(self):
                          'Group Name'
                          '&fq=is_annotation%3Afalse'
                          '&start=0'
-                         '&rows=10000000'
+                         '&rows={}'
                          '&q=django_ct%3Adata_set_manager.node'
                          '&wt=json'
                          '&facet=true'
                          '&facet.limit=-1'.format(
-                                 self.valid_uuid))
+                             self.valid_uuid, constants.REFINERY_SOLR_DOC_LIMIT
```

Comment:
Reverse?

Reply:
This is what we want here (rows corresponding to the REFINERY_SOLR_DOC_LIMIT)